packages/app/src/components/pages/Home/ApplicationViewByStages.js

```diff
@@ -17,35 +22,65 @@ const ApplicationViewByStages = () => {
     return statuses.map( (status, index) => <option key={index} selected={status === currentlyViewing} value={status}>{status}</option> )
   }
 
+  const renderSortArrow = (columnName) => {
+    if (!sortConfig) {
+      return
+    }
+    return sortConfig.key === columnName ?
+      <FontAwesomeIcon icon={sortConfig.direction === 'ascending' ? "arrow-down" : "arrow-up"} />
+      : null
+  }
```

Comment:
Nitpick: I think that first early return is unnecessary. This is a little cleaner:
```suggestion
  const renderSortArrow = (columnName) => sortConfig && sortConfig.key === columnName
    ? <FontAwesomeIcon icon={sortConfig.direction === 'ascending' ? "arrow-down" : "arrow-up"} />
    : null
```